### DISCUSSION TITLE
How can I determine whether a field has been modified but returned to its default value?

### DISCUSSION THREAD

**Discussion by Nfinished** [react-hook-form/react-hook-form#10839] [Q&A]
It seems like `isDirty` is falsy if a form field with a default value is modified and then returned to its default value by user interaction. How can I detect this? I don't think I can used `isTouched` because that'll just become true on any blur event even if there wasn't modification.

**Comment by Moshyfawn** [comment]
How do you "return" the field to it's default value? Via the [`reset`](https://react-hook-form.com/docs/useform/reset) or [`resetField`](https://react-hook-form.com/docs/useform/resetfield) form methods?

**Reply by Nfinished** [reply]
This would be in the case of a user manually re-entering the initial value
